manim/scene/scene.py

```diff
@@ -873,7 +880,6 @@ def handle_play_like_call(func):
         """
 
         def wrapper(self, *args, **kwargs):
-            self.update_skipping_status()
```

Comment:
Can you please explain why it is not necessary here to update the skipping status?

Reply:
Because now it will be done before, in `handle_caching_play` or `handle_caching_wait` !